### PR TITLE
Adding fastverse R package.

### PR DESCRIPTION
**Pull request by SebKrantz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by HeavyTony2** [comment]
R-Kit isn't on Conda-Forge https://github.com/conda-forge/staged-recipes/pull/17458/files

**Comment by SebKrantz** [comment]
Ok, I see you are working this out? Let me know if I should do anything.

**Comment by mfansler** [comment]
@SebKrantz I'm working on it. I had it on hold to figure out why the OSX builds don't use OpenMP, but that seems now like an upstream issue (https://github.com/2005m/kit/issues/26). Since this is dependent, I'll take it off hold and just make an issue once the feedstock is generated. If you have insight, feel free to chime in; otherwise, I'll ping when `r-kit` is available through the channel.

**Comment by SebKrantz** [comment]
@mfansler the kit author is right in saying that OpenMP is not by default available for Mac, but the package is on CRAN where it is also built without OpenMP for Mac, and it is also on [r-universe](https://fastverse.r-universe.dev/ui#package:kit) which also builds binaries for Mac. `src/kit.h` also begins with `#ifdef _OPENMP`, so building on a system without OpenMP should not be a problem. So I think this has to do with conda-forge's build infrastructure. If this is very complicated to sort out I'm fine with not publishing this package to conda-forge, but let me know if you can sort it.

**Comment by mfansler** [comment]
@SebKrantz thanks. Yeah, it's not a problem - it builds perfectly fine without it. It was just that when I did try including `openmp` in the **osx** builds, the configure script provided by `kit` didn't recognize it.

**Comment by dbast** [comment]
There are no compilers specified in the recipe... cran meta data says `NeedsCompilation: yes`. The fastverse repo contains to be compiled code https://github.com/fastverse/fastverse/blob/main/src/attribute_handling.c ... so compilers and build tools have to be added to the build section?

**Comment by mfansler** [comment]
CRAN was down when CI ran - will need a @conda-forge-admin, please restart ci

File: recipes/r-fastverse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    - r-matrixstats
+
+test:
+  commands:
+    - $R -e "library('fastverse')"           # [not win]
+    - "\"%R%\" -e \"library('fastverse')\""  # [win]
+
+about:
+  home: https://sebkrantz.github.io/fastverse/, https://github.com/SebKrantz/fastverse
+  license: GPL-3
```
Comment (dbast) [inline recipes/r-fastverse/meta.yaml:53]: According to CRAN metadata `# License: GPL-3`:
```suggestion
  license: GPL-3.0-only
```

**Pull request merged**